flow-nodes/api-builder-plugin-petstore/test/test.js

```diff
@@ -1,64 +1,45 @@
 const { expect } = require('chai');
-const { MockRuntime } = require('@axway/api-builder-sdk');
+const { MockRuntime } = require('@axway/api-builder-test-utils');
 
-const getPlugin = require('./index');
-
-function getLogger () {
-	const logger = {
-		debug: () => {},
-		trace: () => {},
-		info: () => {},
-		warn: () => {},
-		error: () => {},
-		fatal: () => {}
-	};
-	logger.scope = () => logger;
-	return logger;
-}
+const getPlugin = require('../index');
 
 describe('flow-node petstore', () => {
-	let runtime;
+	let plugin;
+	let flowNode;
 	before(async () => {
```

Comment:
we changed the plugin test template to use beforeEach ... you might switch it here as well and this is valid comment for all other plugins